components/Wallet/Send.js/GasCustomization.jsx

```diff
@@ -89,11 +92,14 @@ const GasCustomization = ({
             const gas = await estimateGas(newGasPrice)
             setEstimatedGas(gas)
           }}
-          variant={
-            gasToButtonMap(gasLimitLocal, gasPriceLocal) === 'medium'
-              ? 'tertiary-selected'
-              : 'tertiary'
-          }
+          // variant={
```

Comment:
Hey @as-dr i'll take of this for now, but in the future just delete > comment out